### ISSUE TITLE
[Feature Request] Coach select between players passing to get5

### ISSUE THREAD

**Issue by kubo6472** [PhlexPlexico/G5V#58]
I know for some time, that you can make a player on a server with get5 a coach quite easily. Got me wondering. How hard it would be to implement the same kind of toggle the team cap has to also toggle who the coach is?

**Comment by PhlexPlexico** [comment]
I mean once you are in the game, you can simply type !coach to swap to the coach position. I'm not sure if there is a call that can be set like `get5_coach <team> <steamid>`... Something I would have to look into.

**Comment by PhlexPlexico** [comment]
Yeah, much like ESEA I think as soon as you enter the server you just have to type `!coach`, since there does not appear to be a way to set it as a cvar from the brief peeking at the code I've done.

**Comment by kubo6472** [comment]
I can then cross post it as a FR on the get5 if needed. This is inspired by faceit, where you can pre-specify the coach right away.

**Comment by PhlexPlexico** [comment]
That could work - as long as there is a way to set it in the match config, we should be able to add it to the API! That being said, this is more of an API request right now, since it doesn't even exist in there so I am going to just move this issue.

Thanks for the idea!
